### DISCUSSION TITLE
Github Terminated My Copilot Subscription

### DISCUSSION THREAD

**Discussion by sonmez-baris** [community/community#69531] [Copilot Conversations] [answered]
### Select Topic Area

Bug

### Body

Hello;

On January 20, 2023, I bought a 1-year subscription to Github Copilot for $100. Although I had about 3 more months to use it, today I received a message from Github that my subscription has ended. The screenshot of the relevant message and the screenshot of my payment are attached. How can i communicate for this problem?

![copilot_message](https://github.com/community/community/assets/51136048/2caa68b8-2a8a-4714-b400-8b08418a7aa3)
![payment](https://github.com/community/community/assets/51136048/160bf8db-d1dd-40f3-beb8-a620a282b803)

**Comment by davecheney** [comment, marked as answer]
@sonmez-baris there was a problem processing the payment. After several retries if the charge cannot be processed the subscription is cancelled. Please contact support https://support.github.com/contact.

**Reply by sonmez-baris** [reply]
Thank you for your reply. I have written to support, but no payment renewal request has been made for my copilot subscription. This does not appear in my payment history either. Moreover, there was no cancellation in my Github Pro account. I hope Github Support will resolve the issue quickly.

**Comment by sonmez-baris** [comment]
I asked to Github Support 2 days ago but i have not any answer yet. What is the default reply time?
